### PR TITLE
Update versions and fix KES Test

### PR DESCRIPTION
**Pull request by cniackz**
### Objective:

* New Go Version, from 1.19 to 1.20
* Also couple of extra changes for failing tests like KES due to missing policies.

### CODE REVIEW

**Comment by cniackz** [comment]
I don't understand why KES Test is failing here:

```
Added `kestest` successfully.
Handling connection for 9000
mc: <ERROR> Failed to get status information: not authorized: insufficient permissions.
/home/runner/work/operator/operator/testing/console-tenant+kes.sh: cannot mc admin kms key status kestest --insecure
Deleting cluster "kind" ...
E0218 00:18:03.099823   47397 portforward.go:234] lost connection to pod
E0218 00:18:03.105574   41874 portforward.go:234] lost connection to pod
Error: Process completed with exit code 111.
```

If `Added `kestest` successfully.` was added, why we are failing to get the status: `cannot mc admin kms key status kestest --insecure
Deleting cluster "kind" ...`

I am debugging this issue...

**Comment by dvaldivia** [comment]
`tenant-kes` test is failing due to `mc: <ERROR> Failed to get status information: not authorized: insufficient permissions.` error, so I we need to look into it

**Comment by allanrogerr** [comment]
@cniackz @dvaldivia Let us know when to re-review

**Comment by cniackz** [comment]
@allanrogerr I am still trying to understand why KES test is failing here, @kannappanr gave me some ideas, so I will be trying to fix this test prior review but will let you know when it is ready for review, thanks for checking on this! 👍

**Comment by cniackz** [comment]
@dvaldivia, @kannappanr gave me the answer already last Saturday, it was due to missing Policy `/v1/status`, I did the fix already in this PR that will allow us to get the status again, if it pass, I will ping you via Slack to review this together.

**Comment by cniackz** [comment]
image

**Comment by cniackz** [comment]
We should be removing Prometheus and Log test since we are no longer supporting those features. But that can be done later on!.

**Comment by cniackz** [comment]
Guys, this is ready to review all tests are now passing:

Screenshot 2023-02-22 at 6 50 05 PM

**Review by pjuarezd — APPROVED** [review]

File: testing/common.sh [hunk 1]
```diff
@@ -128,9 +125,6 @@ function install_operator_version() {
   # Initialize the MinIO Kubernetes Operator
   kubectl minio init
 
-  echo "Scaling down MinIO Operator Deployment"
-  try kubectl -n minio-operator scale deployment minio-operator --replicas=1
```
Comment (Praveenrajmani) [inline testing/common.sh:132]: Q: Why is this removed?

**Review by Praveenrajmani — DISMISSED** [review]
LGTM

File: testing/common.sh (hunk 1 above)
Comment (cniackz) [reply testing/common.sh:132]: Because it is not needed, we have two Operator replicas and that is perfectly fine and the reason they are reducing this to one is making test more prone to failure as we are not even waiting for the proper termination of this pod. So better to not have this line if our test is not reliable because of it.

**Review by allanrogerr — DISMISSED** [review]

**Review by reivaj05 — DISMISSED** [review]

**Comment by pjuarezd** [comment]
This PR is a nice well needed refresher, on top of it:  can we modify this PR to keep tests for 1.19.x and 1.20.x @cniackz?

Motivations:
* Keep checking on 1.19.x for the near future, when other projects like minio, minio-go, madmin-go, etc go 1.20.x  jump to 1.20.x we are ready.
* 1.19.6 and 1.20.1 [solve a lot of CVE's](https://golangtutorial.dev/news/go-1.20.1-and-go-1.19.6-versions-released/), would be optimal test/build over the latest bugfix releases
* I hope this and other projects are moving soon to 1.20.x

**Review by pjuarezd — APPROVED** [review]

**Review by Praveenrajmani — APPROVED** [review]

**Review by allanrogerr — APPROVED** [review]
